server/src/user/test/user.e2e-spec.ts

```diff
@@ -87,4 +87,42 @@ describe('UserController (e2e)', () => {
       });
     });
   });
+
+  describe('DELETE /me/?houseId=:houseId', () => {
+    // const userId = 106;
+    const houseId = 106;
+
+    it('should return 401 if not authenticated', async () => {
+      await request(app.getHttpServer())
+        .delete(`/me/?houseId=${houseId}`)
+        .expect(401);
+    });
+
+    it('should return 403 if a user is only admin', async () => {
+      await request(app.getHttpServer())
+        .delete(`/me/?houseId=${houseId}`)
+        .set('Cookie', [`token=${token}`, `csrf-token=${csrfToken}`])
+        .set('x-csrf-token', csrfToken)
+        .expect(403);
+    });
+
+    it('should return success message with 200', async () => {
+      const nonAdminUser = await request(app.getHttpServer())
+        .post('/auth/login')
+        .send({
+          name: 'Felip',
```

Comment:
I think you don't need to pass name for login!

Reply:
Fix it.
c7fbf3878fea460517f6ab3c4c2edbbeceb299ca